js/package.json

```diff
@@ -15,5 +15,6 @@
     "lint": "eslint . --ext .ts --ext .tsx",
     "format": "clang-format --glob=\"{common/lib/**/*.ts,node/{lib,script,test}/**/*.ts,node/src/**/*.{cc,h},web/{lib,script,test}/**/*.ts,react_native/{android,example,ios,lib}/**/*.{ts,mm,java}}\" --style=file -i"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {}
```

Comment:
is it required by some toolset to add an entry for "dependencies"?

Reply:
There was a warning message for caniuse-lite package and 'dependencies' is added by a npm automatically.